proof/hash_to_field.saw

```diff
@@ -44,19 +32,14 @@ let hash_to_field_spec nelems aug_len msg_len DST_len = do {
 // Proofs
 ///////////////////////////////////////////////////////////////////////////////
 
-let verify_expand_message_for len_in_bytes aug_len msg_len DST_len = do {
-  block_data_order_ovs <- make_block_data_order_ovs [aug_len, msg_len, eval_size {| DST_len+4 |}];
-  r <- crucible_llvm_verify m "expand_message_xmd" (concat bcopy_ovs (concat block_data_order_ovs [blst_sha256_emit_ov, blst_sha256_hcopy_ov])) false (expand_message_spec len_in_bytes aug_len msg_len DST_len) abc;
-  return r;
+let verify_hash_to_field_for aug_len msg_len DST_len nelems = do {
+  let L = 64; // sizeof(vec384) + 128/8 = 48 + 16
+  let len_in_bytes = eval_size {| nelems * L |};
+  block_data_order_ovs <- make_block_data_order_ovs [aug_len, msg_len, eval_size {| (33+DST_len+1+9)+63 |}]; // last size is b_i_blocks from hash_to_field.c
+  //expand_message_xmd_ov <- verify "expand_message_xmd" (concat bcopy_ovs (concat block_data_order_ovs [blst_sha256_emit_ov, blst_sha256_hcopy_ov])) (expand_message_spec len_in_bytes aug_len msg_len DST_len); // TODO: does not terminate
```

Comment:
what is causing this change?

Reply:
I don't know. It seems that Z3 chokes on it, but abc does fine...